README.md

```diff
@@ -23,6 +23,14 @@ Type the path to the output to run the program.
 
 After running the program generates a file name TimeSheet.txt, this is where you copy the inputed hours from.
 
+## Usage
+
+- The first thing it will ask you is the month. Input the month number and not the name.
```

Comment:
If you added visual examples in addition to what you have it may make it clearer.  Something like -
```
Prompt: Please enter Sundays month(Ex January - 1):  
Enter : 1
```